testkit-backend/backend.go

```diff
@@ -1357,6 +1357,7 @@ func (b *backend) handleRequest(req map[string]any) {
 				"Feature:API:Type.Spatial",
 				"Feature:API:Type.Temporal",
 				"Feature:API:Type.Vector",
+				"Feature:API:Type.UnknownType",
```

Comment:
Feature name and TestKit protocol specs will be adjusted to follow the new naming convention (unsupported instead of unknown).